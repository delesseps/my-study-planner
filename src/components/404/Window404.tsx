import * as React from "react";

interface I404Props {}

const Window404: React.FunctionComponent<I404Props> = props => {
  return <div>404 - Not Found</div>;
};

export default Window404;
