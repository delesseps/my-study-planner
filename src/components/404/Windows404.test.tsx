import React from "react";
import { mount } from "enzyme";

import Window404 from "./Window404";

describe("Window404", () => {
  it("should render correctly", () => {
    const component = mount(<Window404 />);

    expect(component).toMatchSnapshot();
  });
});
