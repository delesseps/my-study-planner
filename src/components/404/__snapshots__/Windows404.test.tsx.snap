// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Window404 should render correctly 1`] = `
<Window404>
  <div>
    404 - Not Found
  </div>
</Window404>
`;
